core/conversion/evaluators/prim.cpp

```diff
@@ -95,6 +95,11 @@ auto prim_registrations =
                             tensor_holder.hold_tensor(args.at(in).ITensor());
                             auto ival = c10::IValue(std::move(c10::make_intrusive<TensorContainer>(tensor_holder)));
                             list.emplace_back(std::move(ival));
+                          } else if (args.at(in).isIValue() && args.at(in).IValue()->isCustomClass()){
+                            auto tensor_holder = TensorContainer();
+                            tensor_holder.hold_tensor(args.at(in).IValue()->toCustomClass<TensorContainer>().get()->tensor());
+                            auto ival = c10::IValue(std::move(c10::make_intrusive<TensorContainer>(tensor_holder)));
+                            list.emplace_back(std::move(ival));
```

Comment:
Can you add a test case for this covering aten::chunk ? Please refer to our converter test suite https://github.com/NVIDIA/TRTorch/tree/master/tests/core/conversion/converters